src/main/io/displayport_msp_bf_compat.c

```diff
@@ -319,25 +319,21 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_AUTO_THR1:
         return BF_SYM_AUTO_THR1;
+        */
 
     case SYM_ROLL_LEFT:
-        return BF_SYM_ROLL_LEFT;
-
     case SYM_ROLL_LEVEL:
-        return BF_SYM_ROLL_LEVEL;
-
     case SYM_ROLL_RIGHT:
-        return BF_SYM_ROLL_RIGHT;
+        return BF_SYM_ROLL;
 
     case SYM_PITCH_UP:
-        return BF_SYM_PITCH_UP;
-
     case SYM_PITCH_DOWN:
-        return BF_SYM_PITCH_DOWN;
+        return BF_SYM_PITCH;
```

Comment:
This is ambiguous. DJI should fix the font support.

Reply:
@mmosca Yes, but this patch should better than '?' since we can't push DJI to do this.

Reply:
Not really. Ambiguity can be worse than a question mark.
You will never get full coverage of the OSD with O3, unless DJI implements support for INAV's font.